UnitTestFramework/UnitTestFramework.pq

```diff
@@ -30,7 +30,7 @@ shared Facts = (_subject as text, _predicates as list) =>
 
 shared Facts.Summarize = (_facts as list) as table =>
     [
-        shortOutput = Text.Lower(Environment.FeatureSwitch("short", "true")) = "true",
```

Comment:
IMHO, the name can stay shortOutput. It is more descriptive.

Reply:
ACK.